peertalk/PTUSBHub.m

```diff
@@ -211,7 +211,7 @@ - (void)listenOnQueue:(dispatch_queue_t)queue onStart:(void(^)(NSError*))onStart
 }
 
 
-- (void)connectToDevice:(NSNumber*)deviceID port:(int)port onStart:(void(^)(NSError*, dispatch_io_t))onStart onEnd:(void(^)(NSError*))onEnd {
+- (void)connectToDevice:(NSNumber*)deviceID port:(uint32_t)port onStart:(void(^)(NSError*, dispatch_io_t))onStart onEnd:(void(^)(NSError*))onEnd {
```

Comment:
If you change the type to uint32_t (why not `unsigned int`?) you'll also need to change the call to NSNumber to use `numberWithUnsignedInt` (with cast to uint32_t if you insist on using uint32_t.)

Was there an issue with using `int` here?